src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -994,7 +997,7 @@ function GlobeControls(view, target, domElement, viewerDiv, radius, getPickingPo
         if (this.enableDamping) {
             if (state === CONTROL_STATE.ORBIT && (sphericalDelta.theta > EPS || sphericalDelta.phi > EPS)) {
                 player.play(dampingOrbitalMvt).then(() => this.resetControls());
-            } else if (state === CONTROL_STATE.MOVE_GLOBE && lastRotation.length === 2 && !(lastRotation[1].equals(lastRotation[0]))) {
+            } else if (state === CONTROL_STATE.MOVE_GLOBE && lastRotation.length === 2 && (Date.now() - lastTimeMouseMove < 50) && !(lastRotation[1].equals(lastRotation[0]))) {
```

Comment:
While I approve the idea of moving this to a separate function, I'd prefer a more concise version:

```js
function shouldAnimate() {
  // animate only if XXXXX and YYY
  return (Date.now() - lastTimeMouseMove < 50) && !(lastRotation[1].equals(lastRotation[0]));
}
```
But well, @nosy-b its your code, so as long there's a comment you can write it the way you like :)